.github/dependabot.yml

```diff
@@ -9,3 +9,8 @@ updates:
   - dependency-name: rubocop
     versions:
     - "> 0.49.1"
+  - package-ecosystem: "github-actions"
```

Comment:
Thanks for the contribution! This indentation looks incorrect, though. Should this section be outdented one level so that it is at the same level as `- package-ecosystem: bundler`?

Reply:
Thanks, Took care of it. PTAL.